data/fiberids/grandfibermap.20210314.txt

```diff
@@ -0,0 +1,2599 @@
+\cob  fld   cf mod  mf   cm cbrd  brd mtp      x        y       r   sp  fh  sfib  fiberid  sunss   id(A)        id(C)        id(BA)       id(BC)
+ nan  nan  nan nan nan  nan nan nan  nan      nan      nan     nan  1   1   eng       1    emp  EN-1-1-1-x   nan          nan          nan         
```

Comment:
`nan` values look strange. Do they read cleanly? Can we write something else? A dash, or ellipsis, perhaps?

Reply:
Thank you for suggestion. I'll use dash.